fieldservice_sale_recurring/models/sale_order_line.py

```diff
@@ -38,7 +38,7 @@ def _field_create_fsm_recurring_prepare_values(self):
 
     def _field_create_fsm_recurring(self):
         """Generate fsm_recurring for the given so line, and link it.
-        :return a mapping with the so line id and its linked fsm_recurring
+        return a mapping with the so line id and its linked fsm_recurring
```

Comment:
this is normal when we describe input/output parameters/variables, leave its as it's pls